### PR TITLE
Fix wrong uniform names

### PR DESCRIPTION
**Pull request by System98**
- Fix wrong color names in german uniform names
- Fix wrong color names in english uniform names

### CODE REVIEW

**Comment by System98** [comment]
Please also check/change the classnames JJ. :)

File: addons/units/stringtable.xml [hunk 1]
```diff
@@ -126,8 +126,8 @@
             <German>Kampfkleidung, langärmelig (schwarzes Hemd &amp; grüne Hose &amp; schwarze Stiefel)</German>
         </Key>
         <Key ID="STR_TACS_Units_Uniform_Garment_LS_BS_EP_TB">
-            <English>Combat Garment Long Sleeves (Black Shirt &amp; Grey Pants &amp; Tan Boots)</English>
-            <German>Kampfkleidung, langärmelig (schwarzes Hemd &amp; graue Hose &amp; hellbraune Stiefel)</German>
+            <English>Combat Garment Long Sleeves (Brown Shirt &amp; Grey Pants &amp; Tan Boots)</English>
```
Comment (jonpas) [inline addons/units/stringtable.xml]: Self-note, classname change.

File: addons/units/stringtable.xml [hunk 2]
```diff
@@ -166,8 +166,8 @@
             <German>Kampfkleidung, langärmelig (hellbraunes Hemd &amp; grüne Hose &amp; schwarze Stiefel)</German>
         </Key>
         <Key ID="STR_TACS_Units_Uniform_Garment_LS_BS_TP_TB">
-            <English>Combat Garment Long Sleeves (Black Shirt &amp; Tan Pants &amp; Tan Boots)</English>
-            <German>Kampfkleidung, langärmelig (schwarzes Hemd &amp; hellbraune Hose &amp; hellbraune Stiefel)</German>
+            <English>Combat Garment Long Sleeves (Brown Shirt &amp; Tan Pants &amp; Tan Boots)</English>
```
Comment (jonpas) [inline addons/units/stringtable.xml]: Self-note, classname change.

File: addons/units/stringtable.xml [hunk 3]
```diff
@@ -182,8 +182,8 @@
             <German>Kampfkleidung, hochgekrempelt (schwarzes Hemd &amp; grüne Hose &amp; schwarze Stiefel)</German>
         </Key>
         <Key ID="STR_TACS_Units_Uniform_Garment_RS_BS_EP_TB">
-            <English>Combat Garment Rolled Sleeves (Black Shirt &amp; Grey Pants &amp; Tan Boots)</English>
-            <German>Kampfkleidung, hochgekrempelt (schwarzes Hemd &amp; graue Hose &amp; hellbraune Stiefel)</German>
+            <English>Combat Garment Rolled Sleeves (Brown Shirt &amp; Grey Pants &amp; Tan Boots)</English>
```
Comment (jonpas) [inline addons/units/stringtable.xml]: Self-note, classname change.

File: addons/units/stringtable.xml [hunk 4]
```diff
@@ -222,8 +222,8 @@
             <German>Kampfkleidung, hochgekrempelt (hellbraunes Hemd &amp; grüne Hose &amp; schwarze Stiefel)</German>
         </Key>
         <Key ID="STR_TACS_Units_Uniform_Garment_RS_BS_TP_TB">
-            <English>Combat Garment Rolled Sleeves (Black Shirt &amp; Tan Pants &amp; Tan Boots)</English>
-            <German>Kampfkleidung, hochgekrempelt (schwarzes Hemd &amp; hellbraune Hose &amp; hellbraune Stiefel)</German>
+            <English>Combat Garment Rolled Sleeves (Brown Shirt &amp; Tan Pants &amp; Tan Boots)</English>
```
Comment (jonpas) [inline addons/units/stringtable.xml]: Self-note, classname change.

File: addons/units/stringtable.xml [hunk 5]
```diff
@@ -302,8 +302,8 @@
             <German>T-Shirt &amp; Jeans (schwarzes T-Shirt &amp; hellbraune Hose &amp; schwarze Stiefel)</German>
         </Key>
         <Key ID="STR_TACS_Units_Uniform_TShirt_JP_LS_TP_BB">
-            <English>T-Shirt &amp; Jeans (Blue Shirt &amp; Tan Pants &amp; Black Boots)</English>
-            <German>T-Shirt &amp; Jeans (blaues T-Shirt &amp; hellbraune Hose &amp; schwarze Stiefel)</German>
+            <English>T-Shirt &amp; Jeans (Blue Shirt &amp; Blue Pants &amp; Black Boots)</English>
```
Comment (jonpas) [inline addons/units/stringtable.xml]: Self-note, classname change: `LP`.

**Comment by jonpas** [comment]
Fixes #28

**Pull request merged**